### ISSUE TITLE
`vater sync` causes permission denied error 

### ISSUE THREAD

**Issue by marissaeinhorn** [uwardlaw/vater#164]
`vater sync` executes with the following error
```
control@control-einhorn:~/vater$ vater sync
Traceback (most recent call last):
  File "/home/control/vater/control-services/cli/vater.py", line 137, in <module>
    main()
  File "/home/control/vater/control-services/cli/vater.py", line 11, in main
    c = Config(p.args.configPath, p.args.envPath)
  File "/home/control/vater/control-services/cli/config.py", line 257, in __init__
    self.setupDataFolders()
  File "/home/control/vater/control-services/cli/config.py", line 268, in setupDataFolders
    os.makedirs(dirPath)
  File "/usr/lib/python3.8/os.py", line 223, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/home/control/vater/control-services/data/jenkins'
```

In `config.py` the code searches for service directories and creates them if they don't exist

**Comment by uwardlaw** [comment]
`config.py` should create the folders

**Comment by marissaeinhorn** [comment]
Solved by #145
